scripts/bootstrap.sh

```diff
@@ -105,12 +151,13 @@ if ($installing || $upgrading); then
     echo "Bootstrap salt $version"
     wget -O salt_bootstrap.sh https://bootstrap.saltstack.com --no-verbose
 
+    # -x  Changes the Python version used to install Salt.
     # -P  Allow pip based installations.
     # -F  Allow copied files to overwrite existing(config, init.d, etc)
     # -c  Temporary configuration directory
     # -M  Also install master
     # https://github.com/saltstack/salt-bootstrap/blob/develop/bootstrap-salt.sh
-    sh salt_bootstrap.sh -P -F -c /tmp stable "$version"
+    sh salt_bootstrap.sh -x python3 -P -F -c /tmp stable "$version"
```

Comment:
strictly required to roll this out at the same time as the new bootstrap script and its possible actions? Maybe yes because of dependencies, just checking

Reply:
(if not it could be a follow-up PR)

Reply:
That's an interesting point. I could remove that `-x python3` and the salt version bump so the other changes could go through and be tested - I have made *a lot* of them - so that the follow up PR is just two small changes without all the other things that could go wrong.

Reply:
I've reverted those two changes now. This PR can be merged a bit more safely now.